metrophilly/sermon-processor
# feat(10): refactor processing lib to MoviePy

Closes #10 

## Description

This PR should take care of the incorrect mediafile generation by dumping the temp folder each run. It also adds:
- refactoring the video encoding to use `moviepy` instead of `ffmpeg` to apply crossfades, which should just run the encoding in a single pass now
- shifting the intro and outro.mp4's to the data file to allow us delete the `tmp`dir without incurring a re-download of those files (which would almost never be intentional)

## How to Test

Run the script as usual for both audio and video processes and see that it generates as expected.
